cluster-autoscaler/core/scale_up.go

```diff
@@ -224,6 +224,9 @@ func ScaleUp(context *AutoscalingContext, unschedulablePods []*apiv1.Pod, nodes
 					nodeInfos[newId] = nodeInfos[oldId]
 					delete(nodeInfos, oldId)
 				}
+				context.LogRecorder.Eventf(apiv1.EventTypeNormal, "CreatedNodeGroup",
+					"NodeAutoprovisioning: create new node group %v", newId)
```

Comment:
I'd keep time consistent, you have both deleted and create, should it be past or present?

Reply:
+1, also "NodeGroupCreated" would be more consistent with other events in Kubernetes (like "PodScheduled")

Reply:
fixed tense in message to be consistent